gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-2-entrypoints/step-2-entrypoints-0-architecture.component.html

```diff
@@ -33,18 +33,18 @@
           <!-- Note for migration: Angular Material 15 automatically add radio -->
           <mat-radio-button [checked]="syncOption?.selected"></mat-radio-button>
           <gio-selection-list-option-layout>
-            <gio-selection-list-option-layout-title> HTTP proxy </gio-selection-list-option-layout-title>
+            <gio-selection-list-option-layout-title>{{ httpProxyEntrypoint?.name }}</gio-selection-list-option-layout-title>
 
             <gio-selection-list-option-layout-body>
-              <div>Web APIs can be accessed using the HTTP protocol. Defines endpoints, valid request and response formats.</div>
-              <div>
-                Such as:
-                <span class="gio-badge-neutral">REST API (HTTP)</span>
-                <span class="gio-badge-neutral">WebSocket Proxy</span>
-                <span class="gio-badge-neutral">SOAP API</span>
-                <span class="gio-badge-neutral">gRPC Proxy</span>
-              </div>
+              <div>{{ httpProxyEntrypoint?.description }}</div>
             </gio-selection-list-option-layout-body>
+
+            <gio-selection-list-option-layout-action>
+              <button type="button" mat-stroked-button (click)="onMoreInfoClick($event, httpProxyEntrypoint)">
+                <mat-icon svgIcon="gio:eye-empty"></mat-icon>
+                More information
```

Comment:
I discussed it with Audrey and apparently, we should not show the `More information` button on that page as it's weird to have it only on the first element.

Reply:
Well we have it for all entrypoint it's really "zarb" to restrict the user with the principle of it's not symetric no ?